packages/components/src/Radio/Radio.tsx

```diff
@@ -101,28 +63,26 @@ export const Radio: React.FC<RadioProps> = ({
   error,
   errorMessage,
   ...props
-}) => {
-  if (label) {
-    return (
-      <>
-        <Label
-          htmlFor={props.id}
-          required={required}
-          variant={labelVariant}
-          color={props.disabled ? 'disabled' : 'text'}
-        >
-          <Box as={RadioInput} pr="8px" error={error} {...props} />
-          {label}
-        </Label>
-        {error && errorMessage && (
-          <ValidationMessage>
-            <Exclamation size={16} />
-            {errorMessage}
-          </ValidationMessage>
-        )}
-      </>
-    );
-  }
-
-  return <RadioInput {...props} />;
-};
+}) => (
+  <>
+    <Box
+      as={Label}
```

Comment:
Why no `<Label>` ?

Reply:
because of the __baseCSS prop

Reply:
I don't see where the difference between a hover effect and the color is. Setting the color like you did will override anything. Why not using `css` prop and do this with the cursor too? :)

Reply:
Yes, I can use cursor and color in the css prop. Then we wouldn't need the color prop on the Label.
At the end we have also Box as label.
```
 <Box
      as={Label}
      htmlFor={props.id}
      required={required}
      variant={`label.${labelVariant}`}
      css={
        props.disabled
          ? { color: 'disabled', ':hover': { cursor: 'not-allowed' } }
          : { color: 'text', ':hover': { cursor: 'pointer' } }
      }
    >
```

Reply:
Ah right. No CSS prop on the label. My bad, sorry.

Makes me think of we should expose the css prop on more components actually 😄 but this is another issue